### PR TITLE
Distance of MS2 to apex

### PR DESCRIPTION
**Pull request by robinschmid**

### CODE REVIEW

File: src/main/java/io/github/mzmine/datamodel/features/types/numbers/RtMs2ApexDistanceType.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  @NotNull
+  @Override
+  public List<RowBinding> createDefaultRowBindings() {
+    return List.of();
+  }
+
+  @Override
+  public boolean getDefaultVisibility() {
+    return true;
```
Comment (SteffenHeu) [inline src/main/java/io/github/mzmine/datamodel/features/types/numbers/RtMs2ApexDistanceType.java:75]: No :D

**Comment by SteffenHeu** [comment]
What's the purpose of this metric?

File: src/main/java/io/github/mzmine/datamodel/features/types/numbers/RtMs2ApexDistanceType.java (hunk 1 above)
Comment (robinschmid) [reply src/main/java/io/github/mzmine/datamodel/features/types/numbers/RtMs2ApexDistanceType.java:75]: That was a copy paste

**Comment by robinschmid** [comment]
Danny needs it to estimate how well the methods work - so how close it triggers to the apex of the peak.
I think this might be interesting overall. Could be some quality criteria for us

**Pull request merged**